src/js/modules/infragistics.ui.editors.js

```diff
@@ -7736,6 +7747,9 @@
 
 			// RegEx for /Date(milisecond)/
 			this._mvcDateRegex = /^\/Date\((.*?)\)\/$/i;
+			if (offset !== null && (offset > 840 || offset < -720)) {
+				console.log($.ig.Editor.locale.dateEditorOffsetRange);
```

Comment:
@Lipata , I discussed this with @kdinev. 
Can we throw a configuration error if displayTimeOffset is outside the valid boundaries?